packages/examples/core/primitives/polyhedron.js

```diff
@@ -0,0 +1,27 @@
+/*
+// title       : Polyhedron
+// author      : Rene K. Mueller
+// license     : MIT License
+// description : Demonstrating the polyhedron() function
+// tags        : polyhedron, geom3, modelling
```

Comment:
i'm not going to complain but just wanted to point out that the tags are pretty inconsistent across the examples.
maybe the README should explain something about these special headers?

Reply:
I'll revisit and build consistency in the examples menu PR